PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/ConnectionStringCheck.java

```diff
@@ -0,0 +1,44 @@
+package com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool;
+
+import com.intellij.codeInspection.*;
+import com.intellij.psi.*;
```

Comment:
We avoid using import statements that end with a star (`*`) so that we don't import more classes than those we need. If I'm not mistaken, this is a setting you can change in IntelliJ itself.

Reply:
yes I made the config in settings thanks!

Reply:
This still needs to be resolved.

Reply:
resolved now